support.md

```diff
@@ -4,9 +4,6 @@ redirect_from: /issues/index.html
 excerpt: "Have a problem or question? We are here to help - but you have to help us help you. We keep track of problems, suggestions and questions about ev3dev using GitHub Issues. This lets us keep everything in one place."
 ---
 
-* Table of Contents
```

Comment:
It was rendering incorrectly in local builds for this particular page and I didn't see an obvious benefit. Looking at it more closely, I think it would be beneficial to keep it, so I'll see if I can get it working properly.